### PR TITLE
Controllers tests: session controller

### PR DESCRIPTION
**Pull request by kuschanton**
First of the controller tests, covers session

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -1,36 +1,11 @@
...
+const PORT = process.env.PORT || 3000
 
 /****************************************************
  Start Server
-****************************************************/
+ ****************************************************/
+
 app.listen(PORT, () => {
-  console.log(`Server running on port:${PORT}`);
-});
```
Comment (kuschanton) [inline index.ts:36]: Had to extract app starting code to a separate file to be able to use app in the test without starting server. Otherwise jest complains about open handles.

**Review by cweems — APPROVED** [review]
Very clean and readable test implementation. It's also a good call to extract the app from index.ts. Looks like no merge conflicts with the utils branch as well.

`setEnvVars` may cause issues in the future because it's setting the environment variables for all tests. E.g. we may need a different way if another test needs to mock a null number pool. No concerns for now though.

**Comment by kuschanton** [comment]
> Very clean and readable test implementation. It's also a good call to extract the app from index.ts. Looks like no merge conflicts with the utils branch as well.
> 
> `setEnvVars` may cause issues in the future because it's setting the environment variables for all tests. E.g. we may need a different way if another test needs to mock a null number pool. No concerns for now though.

I think it's ok to have, for the tests that does not really use any env vars. Those that use them would have to override them anyway.
